lib/src/leak_tracking/retaining_path/_retaining_path.dart

```diff
@@ -125,11 +132,17 @@ Future<List<_ItemInIsolate>> _findClasses(String runtimeClassName) async {
   for (final isolateId in _isolateIds) {
     var classes = await _service.getClassList(isolateId);
 
+    const watingTime = Duration(seconds: 2);
+    final stopwatch = Stopwatch()..start();
```

Comment:
does this stop watch need to be stopped at somepoint?

Reply:
No, it is not doing any background work.